angular/component-model/component-model.js

```diff
@@ -1,6 +1,7 @@
 var ComponentService = angular.module('registry').service("Component", function ($http, $window, $sce, $q) {
 
     function preProcessComponent(component) {
+        console.log("new revision");
```

Comment:
Do we really need this console.log here? If just for debugging purposes, better remove it from the final commit.